From latin1 to utf8

diff -up grep-2.5.1a/AUTHORS.utf8 grep-2.5.1a/AUTHORS
--- grep-2.5.1a/AUTHORS.utf8	2008-10-01 18:42:53.000000000 +0200
+++ grep-2.5.1a/AUTHORS	2008-10-01 18:43:12.000000000 +0200
@@ -40,7 +40,7 @@ send me email.
 
 Alain Magloire maintained GNU grep until version 2.5e.
 
-Bernhard "Bero" Rosenkr‰nzer <bero@arklinux.org> maintained GNU grep until
+Bernhard "Bero" Rosenkr√§nzer <bero@arklinux.org> maintained GNU grep until
 version 2.5.1, ie. from Sep 2001 till 2003.
 
 Stepan Kasal <kasal@ucw.cz> maintains GNU grep since Feb 2004.
